src/components/Post.tsx

```diff
@@ -12,16 +18,19 @@ import { MARKDOWN, PLAIN } from '../utils/constants';
 */
 
 type PostProps = {
```

Comment:
Nitpicky thing but why are you changing the name of the variables? I am afraid by doing so something will break

Reply:
I changed the variable names because id was ambiguous and somewhat vague since we have author id and post id.

Reply:
okay!